test/tests/vectorpostprocessors/csv_reader/transfer/master.i

```diff
@@ -0,0 +1,71 @@
+[Mesh]
+  type = GeneratedMesh
+  parallel_type = 'replicated'
+  dim = 2
+  nx = 10
+  ny = 10
+[]
+
+[Variables]
+  [./u]
+  [../]
+[]
+
+[Kernels]
+  [./diff]
+    type = Diffusion
+    variable = u
+  [../]
+[]
+
+[BCs]
+  [./left]
+    type = DirichletBC
+    variable = u
+    boundary = left
+    value = 0
+  [../]
+  [./right]
+    type = DirichletBC
+    variable = u
+    boundary = right
+    value = 1
+  [../]
+[]
+
+[MultiApps]
+  [./master]
+    type = FullSolveMultiApp
+    input_files = 'sub.i'
+    execute_on = initial
+  [../]
+[]
+
+[Transfers]
+  [./transfer]
+    type = MultiAppUserObjectTransfer
+    multi_app = master
+    direction = to_multiapp
+    user_object = data
+    variable = aux
+  [../]
+[]
+
+[VectorPostprocessors]
+  [./data]
+    type = CSVReader
+    csv_file = '../example.csv'
```

Comment:
So this file exists up a level. @milljm - is this ok with the pbs tests? It's a working relative path.

Reply:
hmmm nope. We now need to create a symlink to that file:

```conf
[VectorPostprocessors]
  [./data]
    type = CSVReader
    csv_file = 'example.csv'
```

```bash
ln -s ../example.csv .
git add -f example.csv
```

Now that the PBS stuff is in, I should install PBS on all the build machines, so it can be a target we can test for.